docker/build-tools/Dockerfile

```diff
@@ -160,6 +160,7 @@ RUN go install sigs.k8s.io/kubetest2/kubetest2-tester-exec@${KUBETEST2_VERSION}
 
 # go install does not seem to be compatible with this pesky code-generator repo
 # Install the code generator tools, and hopefully only these tools, this way
+RUN GO111MODULE=on go get -ldflags="-s -w" k8s.io/code-generator/cmd/applyconfiguration-gen@kubernetes-${K8S_CODE_GENERATOR_VERSION}
```

Comment:
I don't like this (https://github.com/kubernetes/kubernetes/issues/101384). But we should probably do it anyways :-)

Reply:
FWIW - I think we can use `go install` now. Not a big deal though